hudi-examples/hudi-examples-flink/src/main/java/org/apache/hudi/examples/quickstart/factory/CollectSinkTableFactory.java

```diff
@@ -91,11 +91,11 @@ public Set<ConfigOption<?>> optionalOptions() {
    */
   private static class CollectTableSink implements DynamicTableSink {
 
-    private final TableSchema schema;
+    private final ResolvedSchema schema;
     private final String tableName;
 
     private CollectTableSink(
-        TableSchema schema,
+            ResolvedSchema schema,
         String tableName) {
```

Comment:
Can we fix all the indentations?

Reply:
Thanks for your suggestion, I will modify the code.